src/components/ui/ProgressBar.js

```diff
@@ -0,0 +1,35 @@
+import { React } from 'react'
+import styled from 'styled-components/macro'
+
+import ProgressBarStep from '../ui/ProgressBarStep'
+
+const StyledProgressBar = styled.div`
+  display: flex;
+  justify-content: space-between;
+`
+const getStepStatus = (index, currentStep) => {
+  const refactoredIndex = index
```

Comment:
Is this line still necessary?

Reply:
Made duplicate comment about this LOL, previous comment was for the outdated version where `index + 1`

Reply:
But yeah you can remove this